include/runtime_symbol_lookup.hpp

```diff
@@ -9,26 +9,133 @@
 #include "symbol_map.hpp"
 #include "symbol_table.hpp"
 
+#include "ddres_def.hpp"
+#include <array>
 #include <unordered_map>
 
 namespace ddprof {
 
 class RuntimeSymbolLookup {
 public:
+  struct Stats {
+    uint32_t _nb_jit_reads = {};
+    uint32_t _nb_failed_lookups = {};
+    mutable uint32_t _count = {};
```

Comment:
It's weird to have a mutable field here

Reply:
I'm not sure what I prefer. I adjusted according to your proposal.